homeassistant/components/qvrpro/manifest.json

```
@@ -0,0 +1,8 @@
+{
+  "domain": "qvrpro",
```

Thinking about it now, I would suggest to split this domain up. Please adjust other files, documentation accordingly.
```suggestion
  "domain": "qvr_pro",
```